### ISSUE TITLE
Sponsor the project to support the developer

### ISSUE THREAD

**Issue by trimechee** [mhogomchungu/media-downloader#113]
To show our appreciation and thanks for the awesome Media-Downloader, it will be great idea ti add button which allows to sponsor the project via paypal for example, or add discrete non intrusive ads, and for people who don't want to see ads , add an option to disable ads, thank you!

maybe it's also good to change the name of the software so that it is more easily recognizable and referenced by google in the search results

**Comment by mhogomchungu** [comment]
Yes, the name is a bit too generic and i now think it was a mistake to use this name.

I live in Tanzania and paypal does not work in my country.

Most donations i have received are in bitcoins, there are a few western union transfers and bank transfers and bitcoins seems to be the most efficient.

Ads, i do not think i will put any because people do not like to see ads and the traffic to my projects i think is too small and revenue from ads will be too small to matter.

**Comment by trimechee** [comment]
anyway, thank you very much for all your exceptional work and effort to design this software :)

I think many would agree to support the project either by making a donation or by accepting some rare non-intrusive advertising or displaying some pop-ups and the software could include an option to disable advertising, or put a big "donate" button " on the main window....

many people like to download videos and audio especially for offline use on their smartphone especially to save their internet plan or because the videos are in very high definition and it jerks so downloading is a solution and I think this project will continue to progress and will be even more successful in sha Allah !

And Tanzania is very beautiful contry, congratulaions for this wonderul contry and wonderful project, it is the only rare project that I know that allows to easily integrate aria2, and different plugins and extension!

**Comment by mhogomchungu** [comment]
Closing for lack of activity.
